### ISSUE TITLE
When I click on the eye, `<g>` target fly out of the viewport

### ISSUE THREAD

**Issue by txbo** [APIs-guru/graphql-voyager#76]
> awesome project! 

I use voyager in create-react-app, she is very beautiful and useful when I open her in browser. But when I clicked on the eyes, `<g>` disappeared, I found `<g>` style `transform: matrix()` was wrong.

can anyone help me?  this is my voyager file code.

```js
import React from 'react'
import {Voyager} from 'graphql-voyager'
import fetch from 'isomorphic-fetch'
// import { VoyagerWrapper } from './style'

function introspectionProvider (query) {
  return fetch('http://localhost:3333/graphql', {
    method: 'post',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({query: query})
  }).then(response => response.json())
}

export default function GraphqlVoyager (props) {
  return (
    <Voyager
      introspection={introspectionProvider}
      workerURI={process.env.PUBLIC_URL + '/voyager.worker.js'}
    />
  )
}
```

**Comment by IvanGoncharov** [comment]
@txbo Can you please try to reproduce it using online demo: https://apis.guru/graphql-voyager/

**Comment by txbo** [comment]
@IvanGoncharov yep, My results are working fine [here](https://apis.guru/graphql-voyager/), and I tried the koa server is also pretty good. This is an example [demo](https://github.com/txbo/voyager-demo) of recurring the problem, and [css](https://github.com/txbo/voyager-demo/blob/master/src/App.css).

**Comment by stevefan1999** [comment]
Yeah, somewhat if it was used as a child component that if Voyager doesn't have full control over that rendering element, the offset is completely invalidated, resulting in a huge viewport shift if you click/teleport on to something. 

In other words, it looks like GQLVoyager must be used in `ReactDOM.render(<Voyager ...>, document.getElementById('root'))`, unless we can have observable and responsive viewport.

**Comment by vshelke** [comment]
@stevefan1999-personal I used the `ReactDOM.createPortal(<Voyager ...>, document.getElementById('root'))` and after inspecting its in the root element still there is a viewport shift after clicking the sidebar items.

**Comment by IvanGoncharov** [comment]
@vshelke It looks like we fixed this issue in `v1.0.0-rc.27`.
I tried it on your demo but used `http://swapi.apis.guru` as GraphQL Server and updated all dependencies:
https://github.com/txbo/voyager-demo
